compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/decompose_im2col.mlir

```diff
@@ -0,0 +1,142 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-linalg-ext-decompose-im2col))" --split-input-file %s | FileCheck %s
+
+#map = affine_map<(d0) -> (d0 * 5)>
+#map1 = affine_map<(d0) -> (d0 * 4)>
+#map2 = affine_map<(d0) -> (-d0 + 1024, 5)>
+module {
```

Comment:
fly by comment, just add the op by itself and its decomposition instead of the entire loop.